Dockerfile

```diff
@@ -0,0 +1,95 @@
+# Fázis 1: JANUS COMPILE STAGE
```

Comment:
Ez a file lokális tesztelésre van, és nem függ össze a travis.yml-el. Ha ez igy van, ezt inkább látnám szívesen a localdev repóban. 
Ez a file nagyjából a localdeven ezt a két file-t teszi ki. Duplikátumnak tűnik, ha nincs nyomós érvünk megtartani. 
- https://github.com/TechTeamer/vuer_localdev/blob/a0708c143694a5624d0fe65b7e1a14490ada6fbf/build/service/janus/Dockerfile
- https://github.com/TechTeamer/vuer_localdev/blob/8bb182a04abf1e365620e7946e74af74ba0b2fbf/build/install/install-janus-build-env.sh

Reply:
A saját dolgainkat kellene elöször karban tartani, a localdev-be hasznos lenne pr-t nyitni ha van itt olyan change ami még nincs oda átvezetve.

Reply:
Igen, ez csak a local travis teszteléshez lett létrehozva. Át lehet rakni a localdevbe is és ahogy mondod.
Rendben! Kiveszem innen és megy nemsokára  a PR a localdevbe.